app/main.py

```diff
@@ -16,4 +16,21 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+# Определяем словарь для хранения всех переменных, сгруппированных по типу
+sorted_variables = {
+    'mutable': [
+        my_favourite_films,
+        marks,
+        collection_of_coins,
```

Comment:
The `collection_of_coins` is a set, which is an immutable type in Python. It should be moved to the 'immutable' list instead of 'mutable'.